src/medCore/gui/medAbstractWorkspace.cpp

```diff
@@ -100,16 +101,19 @@ medAbstractWorkspace::medAbstractWorkspace(QWidget *parent) : QObject(parent), d
     d->navigatorToolBox->hide();
     d->selectionToolBox->addWidget(d->navigatorToolBox);
 
-    d->layerListToolBox = new medToolBox;
-    d->layerListToolBox->setTitle("Layer settings");
-    d->layerListToolBox->hide();
-    d->selectionToolBox->addWidget(d->layerListToolBox);
+    d->layersToolBox = new medToolBox;
+    d->layersToolBox->setTitle("Layer settings");
+    d->layersToolBox->hide();
 
-    d->interactorToolBox = new medToolBox;
-    d->interactorToolBox->setTitle("Interactors");
+    d->layerListToolBox = new medToolBox(d->layersToolBox);
```

Comment:
Just for info, you don't need to specify that d->layersToolBox is the parent of layerListToolBox. The parenting will be done automatically in the addWidget below.